<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatBox</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.ico</normaloff>logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(57, 65, 81);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>221</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(114, 129, 161);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>-1</y>
      <width>421</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 222, 222);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>161</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(114, 129, 161);</string>
    </property>
    <property name="text">
     <string>Участники: 0 в сети</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>211</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(114, 129, 161);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>221</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(114, 129, 161);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>533</x>
      <y>400</y>
      <width>86</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(78, 88, 110, 255);</string>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>401</y>
      <width>287</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(78, 88, 110, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Напишите сообщение...</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>421</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0); color: black</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(57, 65, 81);</string>
   </property>
   <widget class="QMenu" name="menuChatBox">
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(57, 65, 81);</string>
    </property>
    <property name="title">
     <string>ChatBox</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menuChatBox"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Изменить тему</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Выйти из аккаунта</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
